lib/python/qmk/commands.py

```diff
@@ -212,13 +212,16 @@ def parse_configurator_json(configurator_file):
         cli.log.error(f'Invalid JSON keymap: {configurator_file} : {e.message}')
         exit(1)
 
-    orig_keyboard = user_keymap['keyboard']
+    keyboard = user_keymap['keyboard']
     aliases = json_load(Path('data/mappings/keyboard_aliases.hjson'))
 
-    if orig_keyboard in aliases:
-        if 'target' in aliases[orig_keyboard]:
-            user_keymap['keyboard'] = aliases[orig_keyboard]['target']
+    while keyboard in aliases:
```

Comment:
Just for clarification, this is now recursive as we have aliases that point to aliases. Walking that chain till we get to what should reflect the latest target in the tree?

Reply:
Correct.

Reply:
Ok. So this avoids us having to update "old" references if a keyboard is moved multiple times.